src/hooks/worlds/useWorlds.ts

```diff
@@ -7,11 +7,10 @@ import { World } from "api/world";
 import { worldConverter } from "utils/converters";
 import { WithId } from "utils/id";
 
-export const useOwnWorlds = (userId?: string): WithId<World>[] => {
+export const useWorlds = (): WithId<World>[] => {
   const firestore = useFirestore();
   const worldsRef = firestore
     .collection("worlds")
-    .where("owners", "array-contains", userId ?? "")
```

Comment:
Why have you refactored this? I feel like I'm missing a motivation for this change

Reply:
Think of this change as of 2 actions: 
1) Create new `useWorlds` hook to fetch all available worlds
2) Deletion of an old `useOwnWorlds` because it was not used anywhere